### ISSUE TITLE
Point with number in photo view includes the current photo arrow / В режиме просмотра фото точка с числом включает текущий снимок показанный отдельной стрелкой.

### ISSUE THREAD

**Issue by kabalin** [PastVu/pastvu#429]
![image](https://user-images.githubusercontent.com/329780/144616869-9b4e1dd2-1e1b-47db-99bd-4c2cd7abf29b.png)

т.е. (2) это на самом деле и текущий снимок с отдельной стрелкой рядом, при увеличении зума это выглядит так:
![image](https://user-images.githubusercontent.com/329780/144617904-ec030210-2949-4e93-89c5-73a6d62c0750.png)

При формировании кластера на стороне клиента в режиме просмотра фото нужно исключать из кластера текущий снимок.

**Comment by circus2271** [comment]
it seems that the bug has dissapeared

![pastvu com_p_1307051](https://user-images.githubusercontent.com/6430448/221179045-d9d2ba89-7608-4cfe-aa5c-4ef8f6974a9e.png)

![pastvu com_p_1307051 (1)](https://user-images.githubusercontent.com/6430448/221179183-c4be10e8-5978-4075-aabc-51a3789c8eb4.png)

![pastvu com_p_1307051 (3)](https://user-images.githubusercontent.com/6430448/221179466-39ab4bd6-4f00-4c34-be5e-b99687de7ae5.png)

**Comment by kabalin** [comment]
Нет, на месте. Смотрите, здесь к примеру (https://pastvu.com/p/1640530):
![image](https://user-images.githubusercontent.com/329780/221191774-e1ddb42d-e02d-414b-afc3-e0f142d42ee6.png)
3 снимка включают и тот, что показан отдельной стрелкой со звездочкой. При приближении:
![image](https://user-images.githubusercontent.com/329780/221192197-74a25e48-51a7-4f76-b1fa-66ceafed5a35.png)
